packages/core/bootstrap/src/lib/external-adapter/requester.ts

```diff
@@ -52,6 +52,19 @@ export class Requester {
         response = await axios(config)
       } catch (error) {
         // Request error
+        if (error.code === 'ECONNABORTED') {
+          // Axios timeout code
+          throw new AdapterError({
+            statusCode: 408,
+            name: 'Request Timeout error',
+            providerStatusCode: error?.response?.status ?? 408,
+            message: error?.message,
+            cause: error,
+            errorResponse: error?.response?.data?.error,
+            url,
```

Comment:
This should be a 504 instead.

From Wikipedia: https://en.wikipedia.org/wiki/List_of_HTTP_status_codes

> 408 Request Timeout
> The server timed out waiting for the request. According to HTTP specifications: "The client did not produce a request within the time that the server was prepared to wait. The client MAY repeat the request without modifications at any later time."

> 504 Gateway Timeout
> The server was acting as a gateway or proxy and did not receive a timely response from the upstream server.